.ruby-version

```diff
@@ -1 +1 @@
-2.7.6
+2.7.4
```

Comment:
I reverted this to `2.7.4` because `2.7.6` isn't installed on the Buildkite VMs yet and `2.7.4` is used consistently across the other apps.

Reply:
Good call.